mfes/content/public/content/preview/coreplugins/org.ekstep.pdfrenderer-1.0/renderer/libs/compatibility.js

```
@@ -0,0 +1,505 @@
+// Object.defineProperty()?
+// Support: Android<4.0, Safari<5.1
+(function checkObjectDefinePropertyCompatibility() {
+  if (typeof Object.defineProperty !== 'undefined') {
+    var definePropertyPossible = true;
+    try {
+      // some browsers (e.g. safari) cannot use defineProperty() on DOM objects
+      // and thus the native version is not sufficient
+      Object.defineProperty(new Image(), 'id', { value: 'test' });
+      // ... another test for android gb browser for non-DOM objects
+      var Test = function Test() {};
+      Test.prototype = { get id() { } };
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)